File: Models/EnrichmentStatus.cs
namespace MostaqlK.Models;

/// <summary>
/// Enrichment lifecycle state of a discovered project: whether details have been
/// successfully fetched, are still queued/in-flight, or failed permanently.
/// </summary>
public enum EnrichmentStatus
{
    Pending,
    Enriched,
    Failed
}

File: .repertoire/python/src/mostaql/models/enrichment_status.py
"""Enrichment lifecycle state (C# Models/EnrichmentStatus).

String values match the SQLite ``enrichment_status`` column values written by
the C# app ("Pending"/"Enriched"; "Failed" is never persisted today).
"""

from enum import StrEnum

__all__ = ["EnrichmentStatus"]


class EnrichmentStatus(StrEnum):
    """Lifecycle state of a discovered project's enrichment."""

    PENDING = "Pending"
    ENRICHED = "Enriched"
    FAILED = "Failed"
